.github/workflows/.windows.yml

```diff
@@ -1,5 +1,5 @@
 # reusable workflow
-name: windows
+name: .windows
```

Comment:
name should reflect workflow filename otherwise we end up with orphan workflows in actions tab: https://github.com/moby/moby/actions

![image](https://user-images.githubusercontent.com/1951866/187433480-352fc11f-0037-48f7-adc9-4a3dc6f3c2b5.png)

Reply:
Didn't this change just move it to the top? https://github.com/moby/moby/actions/workflows/.windows.yml :grimacing: